src/post/post.repository.knex.ts

```diff
@@ -152,4 +154,92 @@ export class KnexPostRepository {
             throw err;
         }
     }
+
+    async updateReplyCount(postId: number, increment = true): Promise<void> {
```

Comment:
Added this param increment so that we can reuse this function fore delete later.